internal/pkg/cache/cache.go

```diff
@@ -256,9 +256,12 @@ func New(cfg Config) (h *Handle, err error) {
 	}
 	h.parentDir = parentDir
 
+	// If we can't access the parent of the cache directory then don't use the
+	// cache.
 	ep, err := fs.FirstExistingParent(parentDir)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get first existing parent of cache directory: %v", err)
+		h.disabled = true
```

Comment:
Should we display a warning here or a warning when `SINGULARITY_CACHEDIR` is set to something different than the default ?

Reply:
I think we've been to and fro over warnings here... some users don't like them, some do. I think a warning would be good and will add one.